Translations/make_translation.py

```diff
@@ -1263,18 +1263,56 @@ def get_translation_sanity_checks_text(defs: dict) -> str:
     return sanity_checks_text
 
 
+def get_version_suffix(ver) -> str:
+    suffix = str("")
+    try:
+        # Use commands _hoping_ they won't be too new for one environments nor deprecated for another ones:
+        ## - get commit id; --short=8 - the shorted hash with 8 digits (increase/decrease if needed!)
+        sha_id = f"{subprocess.check_output(['git', 'rev-parse', '--short=8', 'HEAD']).strip().decode('ascii').upper()}"
```

Comment:
Took some liberty to increase SHA commit ID letters since there is a place for at least one more char on a device anyway.